packages/account-abstraction/src/ERC4337EthersProvider.ts

```diff
@@ -198,7 +201,7 @@ export class ERC4337EthersProvider extends BaseProvider {
       wait: async (confirmations?: number): Promise<TransactionReceipt> => {
         console.log(confirmations)
         const transactionReceipt = waitPromise.then((receipt: any) => {
-          console.log('received tx receipt ', transactionReceipt)
```

Comment:
console.log should not be used directly… instead create a separate method in util package and use a debug flag value passed in constructor to control printing of logs

Reply:
will be doing this in separate chores branch